scripts/zh-cn.json

```diff
@@ -0,0 +1,81 @@
+  "3_1_train_scanUsers-0-0": "干得还行，目前为止你还没让我失望.我希望你能继续这么有效率地完成我的要求.你现在需要学会怎么使用这些工具中的其中之一，也就是 <cmd>scanUsers</cmd>，它会检测目标服务器上能供你使用的用户账户.在攻击我们的目标之前，你试验一下这家伙能不能正确工作.你可记住了，我不想看到你犯任何错误.用之前你拼凑出来的那个服务器列表试试手吧.",
+  "3_1_train_scanUsers-1-0": "扫描系统用户，在禁用防火墙之后，使用指令 <cmd>scanUsers address</cmd>.",
+  "3_2_attack_intro_end-0-0": "你学会怎么用了？行，我们可以开始做正事了.",
+  "3_2_attack_intro_end-1-0": "记住，这事没有任何容错.我希望你的名声不是弄虚作假的.那我们开始干活！",
+  "4_0_diversion_intro-0-0": "要辅助我们的攻击，你需要黑进一些服务器来搭建僵尸进程去分流.如果你不精准地遵循我的指令的话，你就会被他们抓住.成功连接 <ip>{{ip}}</ip> 之后我就会跟你解释怎么创建僵尸进程.",
+  "4_0_diversion_intro-1-0": "搞快点！我不想等你磨蹭！",
+  "4_1_diversion_firstZombie-0-0": "你需要在这个服务器上搭建僵尸程序.这之后系统会启动一个后台进程，在我们发布后续的攻击命令之前都会保持空闲状态.现在可是个重要时刻，你可根本不要期望这个计划出什么岔子.",
+  "4_1_diversion_firstZombie-0-1": "搭建僵尸程序，使用指令 <cmd>zombieMaker</cmd>.",
+  "4_2_diversion_otherZombies-0-0": "重复之前的指令——连接服务器之后使用zombieMaker——来搞这台服务器： <ip>{{ip1}}</ip> 以及这台：<ip>{{ip2}}</ip>.",
+  "5_0_front_attack-0-0": "终于终于，最后一块拼图拼上了.等会我们只要激活这些僵尸，让它们去攻击客户端网站，银行的防火墙就会失效.这就是个完美的计划，你可根本别想着偏离计划.",
+  "5_0_front_attack-1-0": "在宕机时间里，我们就要攻击我们的目标了.",
+  "5_0_front_attack-2-0": "就是现在.准备开始僵尸攻击.瞄准 <ip>{{ip}}</ip>.",
+  "5_0_front_attack-3-0": "开始僵尸攻击，使用指令 <cmd>zombieAttack address</cmd>.",
+  "5_1_front_check-0-0": "停停停.先别做任何事.我需要检查分流是不是正常：",
+  "5_1_front_check-1-0": "好了！开始攻击了，连接服务器 <ip>{{ip}}</ip>. 然后赶快！",
+  "6_0_dmz_intern_attack-0-0": "我们现在在DMZ（隔离区）服务器上.我们现在能穿行在他们的网络架构里，一台一台接近目标数据库服务器.我们没争取到多少时间，所以你得赶快.",
+  "6_0_dmz_intern_attack-1-0": "准备好了？好！先去服务器 <ip>{{ip}}</ip>，这台服务器部署着隔离区保护内网的防火墙.",
+  "6_1_firewall_process-0-0": "好，现在，杀掉进程 <cmd>firewall</cmd>，然后你就能连接服务器地址 <ip>{{ip}}</ip>.",
+  "6_1_firewall_process-1-0": "终止一个进程，使用指令 <cmd>kill processName</cmd>.",
+  "6_2_web_intern-0-0": "你现在在一台内部网页服务器上，还剩一步要做：够到内部数据库服务器，地址是 <ip>{{ip}}</ip>.",
+  "6_3_security_alert-0-0": "已检测到未授权连接！正在采取紧急情况措施.",
+  "6_3_security_alert-disconnected": "连接已超时.",
+  "6_3_security_alert-1-0": "哦不……我来得太晚了……她还是让你攻击了银行.抱歉，我不得不告诉你……你成功不了的.",
+  "6_3_security_alert-2-0": "虽然我来晚了，但我在你行动的时候找到了一些对你有用的信息.我成功找到了勒索你的人的其中一台服务器地址.我猜，你现在应该会想反过来追踪她？",
+  "7_0_first_server-0-0": "我找到的地址是：<ip>{{ip}}</ip>.这个大概不是她的电脑，但是也应该是代理机器了.去找找ssh的登录日志，你可能能找到一些有用的东西.",
+  "7_1_second_server-0-0": "你看到了吗？有很多连接都是这个地址发起的：<ip>{{ip}}</ip>. ——顺藤摸瓜处理这个服务器吧，就用你刚刚对这台服务器的处理方式.",
+  "7_2_third_server-0-0": "又一台代理机器，让我们祈祷 <ip>{{ip}}</ip> 是她的真身吧……",
+  "7_3_conclusion": "太棒了！这就是她的个人电脑！我们抓到她了！",
+  "8_0_blackmailer_computer-0-0": "找找她的文件，我们必须找到更多她的信息……",
+  "8_1_final_talk-0-0": "嗯？这是啥？怎么回事？你们怎么登进我电脑上的？",
+  "8_1_final_talk-1-0": "我不想敲诈你的！都是他逼我的！",
+  "8_1_final_talk-2-0": "都是他！他ob̸́̈́̕͜l̶͓̀̒͌̂͐i̴̧̝̪̇̀̒ …",
+  "8_1_final_talk-3-0": "我们抓到她了！现在，趁她把所有东西销毁之前，把这些信息上传发送给警察吧！我给你准备了个特别指令！使用 <cmd>sendToPolice id.txt</cmd> 一切就结束了.",
+  "8_1_final_talk-4-0": "你在等什么？快点发送啊！",
+  "8_1_final_talk-5-0": "我重新夺回控制了！跟你说话的，怂恿你的人！就是他敲诈的我！用命令 <cmd>exit</cmd> 快逃离这个服务器，然后砸掉你的电脑，趁 hi̴͚͒͝ḻ̶̰̄̏ …",
+  "8_1_final_talk-6-0": "你不会相信那个骗子的吧？",
+  "8_2a_condamn-0-0": "谢天谢地，你帮我擦了屁股！现在，该是你闭嘴的时候了.",
```

Comment:
I'm not sure of this first sentence, is this a chinese idiom ?

Reply:
Yes, it is just like "clean up all the mess" in slang.

Reply:
Perfect, thanks !